docs/new-guide-how-to-kickstart-your-IDP.md

```diff
@@ -0,0 +1,50 @@
+---
+id: new-guide-how-to-kickstart-your-IDP
+title: How to Kickstart Your IDP
+---
+
+:::note What is this guide about?
+This guide shows you how to accelerate your platform engineering by importing pre-built building blocks from meshStack Hub. You'll learn how to quickly provide standardized, secure infrastructure for your teams using community-maintained resources.
+:::
+
+:::tip
+For a hands-on walkthrough of this solution, checkout the interactive demo on Storylane: [View the Interactive Demo](https://app.storylane.io/share/hzzabrqbgthk). This demo will guide you step-by-step through
+importing a Building Block Definition from the meshStack Hub and making it available to your platform users.
+:::
+
+## Prerequisites
+
+- You have a user with workspace owner or admin permissions to access platform builder. 
+- You have credentials to deploy AWS S3 Buckets.
+
+## Step by Step Guide
+
+### 1. Visit meshStack Hub
+
+- Go to [hub.meshcloud.io](https://hub.meshcloud.io) and browse the curated collection of ready-to-use Terraform modules.
+- Locate the [AWS S3 Bucket](https://hub.meshcloud.io/platforms/aws/definitions/aws-s3_bucket) module. Review its description and purpose.
+
+### 2. Import a Building Block Definition
+
+- Choose the *AWS S3 Bucket* module.
+- Click **Import to meshStack**.
+- Enter your meshStack URL if prompted.
+- You'll be redirected to meshPanel and asked where to import the building block definition. Select the workspace for the import.
+- You'll land on the building block definition creation page with pre-populated information.
+
+### 3. Finish Creation of the Building Block Definition
+
+- Complete the pre-populated form.
+- The inputs are automatically imported. Fill in the AWS Secret Key and AWS Access Key ID (from Bitwarden).
```

Comment:
d: from Bitwarden is very meshclud specific. Customers may not have bitwarden. But they may struggle in how to get an AWS Key for this access with the right permissions? Perhaps that should be described in teh prerequisites, that access via a key to S3 must be available?

Reply:
Ups I missed that when I migrated the demo guide. Good catch